src/main/java/org/scijava/console/ConsoleService.java

```diff
@@ -62,6 +63,12 @@
 	/** Notifies listeners of output sent to {@code stdout} or {@code stderr}. */
 	void notifyListeners(OutputEvent event);
 
+	/** Stream that writes to stdout without notifying listeners. */
+	PrintStream getSystemOutBypassListeners();
```

Comment:
I think we will need this. I want the smarter log window to be able to show the informations written to System.out and System.err separately from the log messages. I really want the LogService to write the logs to the terminal without spamming the OutputListeners.